.eslintrc

```diff
@@ -38,7 +37,9 @@
     // https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/require-default-props.md
     "react/require-default-props": "off",
 
-    "no-unused-vars": ["error", { "args": "after-used" }],
+    "no-unused-vars": "off",
```

Comment:
This is a similar pattern to what we do for `no-use-before-define`: disable the ESLint version since it doesn't handle TypeScript properly and then enable the TS-aware version.